### PR TITLE
feat(rust): on `node create` create symlinks to default vault/identity

### PR DESCRIPTION
**Pull request by adrianbenavides**
It includes some TODO's that will be addressed in future PR's, like the `pid`, `version` and `NodeSetupConfig`.

### CODE REVIEW

File: implementations/rust/ockam/ockam_command/src/node/util.rs [hunk 1]
```diff
@@ -105,6 +109,22 @@ pub(super) async fn create_default_identity_if_needed(
 
     cfg.persist_config_updates()?;
 
+    // Store vault and identity in the new directory
+    let vault_name = hex::encode(random::<[u8; 4]>());
```
Comment (polvorin) [inline implementations/rust/ockam/ockam_command/src/node/util.rs:113]: I'm not sure I follow this part,   but it's already different on https://github.com/build-trust/ockam/pull/3866 where the default' symlinks are copied around

**Comment by polvorin** [comment]
@adrianbenavides   I'll be in favor of just closing this one and https://github.com/build-trust/ockam/pull/3865 ,   and merge https://github.com/build-trust/ockam/pull/3866 that contains both already

File: implementations/rust/ockam/ockam_command/src/node/util.rs (hunk 1 above)
Comment (adrianbenavides) [reply implementations/rust/ockam/ockam_command/src/node/util.rs:113]: Yeah exactly, this was a workaround before what I did on https://github.com/build-trust/ockam/pull/3866

**Comment by adrianbenavides** [comment]
Merged through https://github.com/build-trust/ockam/pull/3866

**Pull request closed without merging**